IvanKorzh/HW7/3.py

```diff
@@ -0,0 +1,10 @@
+def calculated_number_count(str1):
```

Comment:
good job
look at this version
def count_symbols_in_row(word):
    result = {}
    for item in word:
        if item in result:
            continue
        else:
            result.update({str(item): word.count(item)})
    return result